Libraries/LibWeb/DOM/Element.h

```diff
@@ -201,7 +201,7 @@ class Element
 
     void run_attribute_change_steps(FlyString const& local_name, Optional<String> const& old_value, Optional<String> const& value, Optional<FlyString> const& namespace_);
 
-    CSS::RequiredInvalidationAfterStyleChange recompute_style();
+    CSS::RequiredInvalidationAfterStyleChange recompute_style(bool& did_change_custom_properties);
```

Comment:
Would it make sense for `did_change_custom_properties` to be a member of `RequiredInvalidationAfterStyleChange` instead?

Reply:
`recompute_style()` is only used from `update_style_recursively()`, which is the only place where it's known how to process `did_change_custom_properties`. Other places that act based on flags in `RequiredInvalidationAfterStyleChange` won't be able to process `did_change_custom_properties`, so I don't see value in putting it there.